code/style.css

```diff
@@ -138,3 +138,29 @@ button:hover {
   opacity: 0.9;
   transition: all 0.2s ease;
 }
+
+.food-select {
+  padding: 16px;
+  font-size: 16px;
+  width: 100%;
+  background: #e5e9ff;
+  color: #2a007c;
+  font-family: "Bree Serif";
+  border: none;
+  border-radius: 4px;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+}
+
+footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 30px;
+}
+
+footer p {
+  color: white;
+  font-weight: 200;
+}
```

Comment:
Your styling is nice and modern 👍🏻 I think you should make your footer stand out a little more 👨🏻‍🎨